src/api/viya/uploadTables.ts

```diff
@@ -0,0 +1,29 @@
+import { prefixMessage } from '@sasjs/utils/error'
+import { RequestClient } from '../../request/RequestClient'
+import { convertToCSV } from '../../utils/convertToCsv'
+
+export async function uploadTables(
```

Comment:
JSDoc:

Tables are uploaded to SAS as specially formatted CSVs - this is more compact than JSON, and easier to read within SAS.

Reply:
This one is also only used internally, but I've added a JSDoc comment.